awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -260,13 +260,21 @@ def make_request(self, method, endpoint, *args, **kwargs):
                 tower_type = response.info().getheader('X-API-Product-Name', None)
                 tower_version = response.info().getheader('X-API-Product-Version', None)
 
-            collection_major = Version(self._COLLECTION_VERSION).version[0]
-            tower_major = Version(tower_version).version[0]
+            parsed_collection_version = Version(self._COLLECTION_VERSION).version
+            parsed_tower_version = Version(tower_version).version
+            if tower_type != 'AWX':
```

Comment:
I think this `'AWX'` has to be lower case (otherwise, the `test_version_warning_strictness_awx` fails)

Reply:
Tower type should be returned as either `AWX` or `Red Hat Ansible Tower`.... there might be another bug somewhere.